src/mame/heathkit/tlb.h

```diff
@@ -33,19 +33,19 @@ class device_heath_tlb_card_interface : public device_interface
 	virtual void serial_in_w(int state) = 0;
 
 	// optional operations
-	virtual void rlsd_in_w(int state) {}
-	virtual void dsr_in_w(int state)  {}
-	virtual void cts_in_w(int state)  {}
+	virtual void rlsd_in_w(int state)           {}
+	virtual void dsr_in_w(int state)            {}
+	virtual void cts_in_w(int state)            {}
 
 	// optional SigmaSet operations
-	virtual void sigma_ctrl_w(uint8_t data) {}
-	virtual uint8_t sigma_ctrl_r() { return 0x00; }
-	virtual uint8_t sigma_video_mem_r() { return 0x00; }
-	virtual void sigma_video_mem_w(uint8_t val) {}
-	virtual void sigma_io_lo_addr_w(uint8_t val) {}
-	virtual void sigma_io_hi_addr_w(uint8_t val) {}
-	virtual void sigma_window_lo_addr_w(uint8_t val) {}
-	virtual void sigma_window_hi_addr_w(uint8_t val) {}
+	virtual void sigma_ctrl_w(u8 data)          {}
+	virtual u8   sigma_ctrl_r()                 { return 0x00; }
+	virtual u8   sigma_video_mem_r()            { return 0x00; }
+	virtual void sigma_video_mem_w(u8 val)      {}
+	virtual void sigma_io_lo_addr_w(u8 val)     {}
+	virtual void sigma_io_hi_addr_w(u8 val)     {}
+	virtual void sigma_window_lo_addr_w(u8 val) {}
+	virtual void sigma_window_hi_addr_w(u8 val) {}
```

Comment:
It’s the same with these – it’s prone to either ending up looking ragged again, or requiring all the lines in the block to change for a small functional change.